### ISSUE TITLE
Getting "Terminating app due to uncaught exception 'NSInvalidArgumentException', reason: '-[DocuSignPOC.AppDelegate window]: unrecognized selector sent to instance 0x600002316820'"

### ISSUE THREAD

**Issue by JeevanEashwar** [docusign/native-ios-sdk#29]
#### What are you trying to do?
Integrate the Docusign cocoapod.
I did a pod install with `pod 'DocuSign'` which installed DocuSign (2.2.5).
My DSM set up is taken as it is from the example app in the cocoapod is as follows
```
var configurations = DSMManager.defaultConfigurations()
configurations[DSM_SETUP_OFFLINE_SIGNING_HIDE_ALERTS_KEY] = DSM_SETUP_TRUE_VALUE

if #available(iOS 11.0, *) {
    configurations[DSM_SETUP_ICLOUD_DOCUMENT_ENABLED] = DSM_SETUP_TRUE_VALUE
} else {
    configurations[DSM_SETUP_ICLOUD_DOCUMENT_ENABLED] = DSM_SETUP_FALSE_VALUE
}
DSMManager.setup(withConfiguration: configurations)
```

After successful login, I am trying to present SendTemplateControllerWithTemplate.
```
private func presentSendTemplate(_ signingMode: DSMSigningMode) -> Void
        {
            let templatesManager = DSMTemplatesManager()
            templatesManager.presentSendTemplateControllerWithTemplate(withId: Constants.kSampleTemplateOne, signingMode: .online, presenting: self, animated: true) { (vc, error) in
                print(error?.localizedDescription)
                print(vc.self)
            }
        }
```
 #### What is happening?
the app crashed with the following error.
<img width="699" alt="Screenshot 2020-05-11 at 12 34 19 PM" src="https://user-images.githubusercontent.com/16348649/81533399-c139a200-9383-11ea-90d1-a8a4aff8443b.png">

#### What is expected?
template controller screen should appear similar to the example application in the cocoapod.

#### Additional details
Xcode: Version 11.4.1 (11E503a)
Simulator: iPhone 11 (13.4.1)
Swift

**Comment by JeevanEashwar** [comment]
The application also crashes with the above error message, when adding recipients in presentComposeEnvelopeViewController.
![Simulator Screen Shot - iPhone 11 - 2020-05-12 at 10 55 32](https://user-images.githubusercontent.com/16348649/81641704-4209a400-943f-11ea-878e-0a5986d145df.png)

**Comment by thomas-dang** [comment]
Hi @JeevanEashwar,

Thank you for reaching out regarding this issue. Our team is looking into it.

In the meantime, it seems this issue may come up for apps that have an iOS Deployment Target set to iOS 13+. In case you need a temporary workaround until we address this issue in our SDK, you can try setting your app's iOS Deployment Target to iOS 12+ for example and seeing if the issue is still there.

Thanks.

**Comment by JeevanEashwar** [comment]
Yeah, I observed the same.
Also, the issue is resolved on iOS 13+, if we add the window instance in the Appdelegate class.
```var window: UIWindow?```

Thanks

**Comment by ashokds** [comment]
This is fixed in v2.3 release.

perform `pod update 'DocuSign'` to fetch `v2.3`.

It should result in: 
`Installing DocuSign 2.3 (was 2.2.5 and source changed to `https://cdn.cocoapods.org/` from `trunk`)`
